README.md

```diff
@@ -1,3 +1,138 @@
+# DirectAvenue UI Project
+
+This project uses **Vite** with **React** and **TypeScript**, configured with **ESLint**, **Prettier**, **Material UI**, **Husky**, and unit testing setup.
+
+### Content:
+
+1. **Technologies Used**: Lists and explains the key technologies and tools used in the project.
+2. **Prerequisites** - Requirements needed to work with this project
+3. **Installation**: Step-by-step instructions to clone the repository, install dependencies, and get started.
+4. **Available Scripts**: All commands you can run to develop, build, format, lint, test, etc.
+5. **Tool Configurations**: Provides details on each major tool included in the project (Vite, React, TypeScript, ESLint, Prettier, Material UI, Husky).
+6. **Project Structure**: Project folder structure.
+7. **Storybook**: Frontend workshop for building UI components and pages in isolation. Thousands of teams use it for UI development, testing, and documentation. It's open source and free.
+
+## Technologies Used
+
+- **Vite**: A fast and optimized bundler for modern web development.
+- **React**: A JavaScript library for building user interfaces.
+- **TypeScript**: A typed superset of JavaScript that provides static types.
+- **Material UI**: A popular React component library that implements Google's Material Design.
+- **ESLint**: A tool for identifying and fixing problems in your JavaScript/TypeScript code.
+- **Prettier**: A code formatter that ensures consistent code style across your project.
+- **Husky**: A tool for managing Git hooks, ensuring code quality before commits and pushes.
+- **Yarn**: A fast and reliable package manager for managing dependencies.
+- **Jest & React Testing Library**: Tools for unit and integration testing.
+
+## Prerequisites
+
+- **Node.js** (version 14 or later)
```

Comment:
I am unsure if we want to use an old version of node js(14) since we are using version #20 as the minimum in the API.